tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -35,8 +34,9 @@
 _CONFIG_EXPORT = parse.ConfigMode.EXPORT
 # udmi present value: points.$name.present_value
 # where name follows: [a-z][a-z0-9]*(_[a-z0-9]+)*
-_UDMI_PRESENT_VALUE_REGEX = \
-  r'^(points.)[a-z][a-z0-9]*(_[a-z0-9]+)*(.present_value)$'
+_UDMI_PRESENT_VALUE_REGEX = (
```

Comment:
formatting, was more readable previously

Reply:
Lines must be separated due to line-too-long error.